src/IO/S3/PocoHTTPClient.cpp

```diff
@@ -177,7 +177,7 @@ namespace
     bool checkRequestCanReturn2xxAndErrorInBody(Aws::Http::HttpRequest & request)
     {
         auto query_params = request.GetQueryStringParameters();
-        if (request.HasHeader("z-amz-copy-source"))
+        if (request.HasHeader("x-amz-copy-source"))
```

Comment:
??? Did they change the format of the header, or what?

Reply:
It's always been `x-amz-copy-source` since at least 2015. See
https://github.com/aws/aws-sdk-cpp/blame/main/aws-cpp-sdk-s3/source/model/CopyObjectRequest.cpp#L176
and
https://github.com/aws/aws-sdk-cpp/blob/5fb5947c201b2fa89eab3f64142f59f0251704ef/aws-cpp-sdk-s3/source/model/CopyObjectRequest.cpp#L108